### DISCUSSION TITLE
Android Google Gboard keyboard covers cursor

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43966] [Mobile] [answered]
I'm using Gboard with GitHub Mobile for Android and when I write things, eventually my cursor will end up beneath the keyboard. This is incredibly painful and frustrating.

Ideally GitHub Mobile should negotiate with the onscreen keyboard to ensure that this doesn't happen 

---

See two pictures from when the cursor ended up below the keyboard:
.![Screenshot_20230111-094636.png](https://user-images.githubusercontent.com/2119212/211836392-48bae1c2-9f16-44c2-87ce-8eb2d8f6d4e4.png)![Screenshot_20230111-094648.png](https://user-images.githubusercontent.com/2119212/211836413-fc60d1c7-b989-4c17-9b4c-3572544c70a7.png)

---

The current line can also end up partially clipped by the keyboard:

![Screenshot_20230111-094716.png](https://user-images.githubusercontent.com/2119212/211836546-8d36c1a9-e28d-4c52-9b20-6e43eeed729c.png)![Screenshot_20230111-094737.png](https://user-images.githubusercontent.com/2119212/211836572-9a1b52fe-2954-4054-b5fa-2a8b597f42ae.png)

It's also possible for the selection widget to end up partially on top of the Gboard, but I think that's an Android bug

Here's a variant where the selection is on top of the format bar:
![Screenshot_20230111-094920.png](https://user-images.githubusercontent.com/2119212/211836829-62edba79-3b97-4745-9d63-020020466f91.png)

And where is somewhat on top of the Gboard:

![Screenshot_20230111-095001.png](https://user-images.githubusercontent.com/2119212/211837144-14223552-9ef4-483f-bf61-9976e6287c1a.png)

**Comment by jacortinas** [comment, marked as answer]
Hi, @jsoref! This is definitely a bug that should be looked into, I'll create an issue for the team to track and fix this. Thank you!
